pom.xml

```diff
@@ -617,6 +617,16 @@
                             <moduleShortName>login</moduleShortName>
                         </configuration>
                     </execution>
+                    <execution>
+                        <id>compile-gwt-harp-support</id>
+                        <goals>
```

Comment:
It's an overkill to have a module for Harp Support page. But I could not quickly make it look similar to MAT in a different way.